test/org/traccar/protocol/NavisProtocolDecoderTest.java

```diff
@@ -11,13 +11,16 @@ public void testDecode() throws Exception {
 
         NavisProtocolDecoder decoder = new NavisProtocolDecoder(null);
 
+        // Handshake
         verifyNull(decoder, binary(
                 "404E5443010000007B000000130044342A3E533A383631373835303035323035303739"));
 
+        // NTCB F5.2 - Single
```

Comment:
Instead of comments, I would recommend splitting code into 3 methods like `testDecodeNtcb`, `testDecodeFlex2` etc.

Reply:
Fixed